Lib/test/test_time.py

```diff
@@ -163,6 +163,12 @@ def test_sleep(self):
         self.assertRaises(ValueError, time.sleep, -1)
         time.sleep(1.2)
 
+    def test_sleep_invalid(self):
+        with self.assertRaisesRegex(
+                TypeError, 'an integer or a float is required \(got type str\)') as te:
```

Comment:
You want this to be a raw (`r''`) string, in order to escape the parens for the regex engine.

Reply:
Yes. I think you are right. It came to me earlier but I forgot to delete it. I updated the PR, thanks.